packages/backend-defaults/src/entrypoints/database/DatabaseManager.ts

```diff
@@ -89,6 +89,27 @@ export class DatabaseManagerImpl {
     return { getClient, migrations: { skip } };
   }
 
+  /**
+   * Method to be called during shutdown to destroy all known connections.
+   */
+  async shutdown(deps: { logger: LoggerService }): Promise<void> {
+    const pluginIds = Array.from(this.databaseCache.keys());
+    await Promise.all(
```

Comment:
Could go for `.allSettled` here too. I see it _should_ not be possible to throw, but you know how it is, when refactors happens this may start to accidentally short circuit